recipe/meta.yaml

```diff
@@ -1,55 +1,52 @@
-{% set version = "3.4.0" %}
+{% set name = "pydap" %}
+{% set version = "3.5" %}
 
 package:
-  name: pydap
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://github.com/pydap/pydap/archive/refs/tags/{{ version }}.tar.gz
-  sha256: c6bded7461ab4db0f94a46441fd114cff0edf327997a3950fd11056132b9c8e2
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pydap-{{ version }}.tar.gz
+  sha256: 0ee6cb7a4892ef22a15c4e2aa58b2656104c542c74faf16c580c56fdf97647ab
 
 build:
-  number: 0
   noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
-  entry_points:
-    - pydap = pydap.wsgi.app:main
-    - dods = pydap.handlers.dap:dump
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
 
 requirements:
   host:
-    - python >=3.7
+    - python >=3.9
+    - setuptools >=64
+    - setuptools-scm >=8
     - pip
   run:
-    - python >=3.7
-    - beautifulsoup4
-    - docopt
-    - jinja2
+    - python >=3.9
     - numpy
     - webob
-    - six >=1.4.0
-    - webob
-    #  net
+    - jinja2
+    - docopt-ng
+    - beautifulsoup4
+    - lxml
     - requests
+    - importlib-metadata
+    - importlib-resources
 
 test:
   imports:
     - pydap
-  requires:
-    - gunicorn  # [not win]
-    - pastedeploy  # [not win]
-    - pip
-  commands:
-    - pydap -h  # [not win]
```

Comment:
Why did you remove the CLI test? Was it removed upstream? Or it was failing for some reason?

Reply:
Hmm I generated this with `grayskull`, and this was done automatically so I am not familiar with what does `-h` flags mean.

Reply:
Grayskull helps but it is not a silver bullet. That only tests the CLI command. `-h` should output the help text.

Reply:
Oh I see. I though `-h` was some conda stuff for installing. It looks like grayskull left the `pydap` server testing.
```
pydap -h 
```
simply prints the `help docstring` when using pydap as a server. Will add those now